packages/components/src/guide/finish-button.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef, useLayoutEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../button';
+
+export default function FinishButton( { className, onClick, children } ) {
+	const button = useRef( null );
+
+	// Focus the button on mount if nothing else is focused. This prevents a
+	// focus loss when the 'Next' button is swapped out.
+	useLayoutEffect( () => {
+		if ( document.activeElement === document.body ) {
```

Comment:
In #20594, I am encountering an issue where we make certain assumptions about what it means for "no focus" to exist. It appears that the value of `activeElement` may differ depending on browser:

>When there is no selection, the active element is the page's `<body>` or `null`.

https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/activeElement

I mention it here, since this may not cover all focus losses, notably those in Internet Explorer.

![image](https://user-images.githubusercontent.com/1779930/75707343-b92afb00-5c8c-11ea-8084-a4b9571e0531.png)

It might be something where we want to provide some utility, e.g. `wp.dom.hasActiveElement`, since it seems to be an easy issue to overlook.

Reply:
Fix at #20599